src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -45,7 +45,8 @@
 #else
 
 #  define mlx5_av_base(_av)         (_av)
-#  define mlx5_av_grh(_av)          (_av)
+#  define mlx5_av_grh(_av)          ((struct mlx5_grh_av *)(((char*)(_av)) + \
```

Comment:
what is reason for this pointer arithmetic? can u pls add comment to explain?

Reply:
direct cast from address of ```reserved``` cause compilation warning
will add comment